cmd/wallet-web/src/router/routes.js

```diff
@@ -82,11 +82,6 @@ export default [
       },
     ],
   },
```

Comment:
just a thought, previously we shared link of wallet to many vendors, teams. Most of them might be still using/bookmarked old URL, can we have common landing page for wrong URLs which has links for sign/signup for now?

Reply:
I guess we have one already, @birtony can you test and confirm 

![image](https://user-images.githubusercontent.com/29631944/128717787-dcc55d22-bd74-4d56-a2df-8d64a885aeb5.png)

Reply:
Yes, we already have that